server/text_generation_server/layers/fp8.py

```diff
@@ -321,6 +357,17 @@ def get_weights_row(self, weights: "Weights", prefix: str):
         w = weights.get_sharded(f"{prefix}.weight", dim=1)
         # FP8 branch
         if w.dtype == torch.float8_e4m3fn:
+            if self.weight_block_size is not None:
+                scale = weights.get_sharded(f"{prefix}.weight_scale_inv", dim=1)
```

Comment:
`weight_scale_inv` is the same as `weight_scale` in other models? (If so, we might want to make the parameter name configurable in the future, with `weigth_scale` as the default.)

Reply:
Yes, decant process is `(128x128 weight block) * weight_scale_in`